samples/python/python_test.go

```diff
@@ -0,0 +1,197 @@
+package ruby_test
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/paketo-buildpacks/occam"
+	"github.com/paketo-community/python/samples/tests"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	. "github.com/onsi/gomega"
+	. "github.com/paketo-buildpacks/occam/matchers"
+)
+
+var builders tests.BuilderFlags
+var suite spec.Suite
+
+func init() {
+	flag.Var(&builders, "name", "the name a builder to test with")
+}
+
+func TestPython(t *testing.T) {
+	Expect := NewWithT(t).Expect
+
+	Expect(len(builders)).NotTo(Equal(0))
+
+	SetDefaultEventuallyTimeout(60 * time.Second)
+
+	suite := spec.New("Python", spec.Parallel(), spec.Report(report.Terminal{}))
```

Comment:
changing `suite := ...` here to `suite = ...` will likely get rid of that linting error